src/test/regress/columnar_am_schedule

```diff
@@ -3,7 +3,6 @@ test: am_load
 test: am_query
 test: am_analyze
 test: am_data_types
-test: am_functions
```

Comment:
Is this on purpose? seems unrelated to the title of the PR

Reply:
ah never mind, looking at the tests I see we were testing some cstorefdw function in this file.